build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -328,6 +334,10 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     file.write("export IGNORE_PROXY_SETTINGS=#{@ignoreProxySettings}\n")
   end
 
+  if @multiline_enabled.strip.casecmp("true") == 0
```

Comment:
have @multiline_enabled=false default, otherwise this can be undefined value if this not defined in the configmap, right?